integration/plugin/testdata/example-plugin/go.mod

```diff
@@ -1,6 +1,8 @@
 module example-plugin
 
-go 1.20
+go 1.21.1
+
+toolchain go1.21.4
```

Comment:
The `toolchain` line could be removed I think, because the `go` line with a proper semver should be used for the toolchain. In this case is irrelevant because is a test file so it could be removed.

Reply:
removed. please check